Java/src/com/gymfox/echoserver/WazzupClient.java

```diff
@@ -0,0 +1,133 @@
+package com.gymfox.echoserver;
+
+import com.gymfox.communication.IPv4Address;
+
+import java.io.*;
+import java.net.Socket;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+import static com.sun.jndi.ldap.LdapCtx.DEFAULT_PORT;
+
+public class WazzupClient {
+    private final static int MAX_PERMISSION_PORT = 65536;
+    private final static int MIN_PERMISSION_PORT = 1024;
```

Comment:
These constants are the same for `WazzupClient` and `EchoServer`. Can put them in separate `abstract final class ServerUtils`